gateway-ha/src/main/java/io/trino/gateway/ha/persistence/dao/Selectors.java

```diff
@@ -13,85 +13,15 @@
  */
 package io.trino.gateway.ha.persistence.dao;
 
-import org.javalite.activejdbc.Model;
-import org.javalite.activejdbc.annotations.BelongsTo;
-import org.javalite.activejdbc.annotations.Cached;
-import org.javalite.activejdbc.annotations.IdName;
-import org.javalite.activejdbc.annotations.Table;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import static io.trino.gateway.ha.router.ResourceGroupsManager.SelectorsDetail;
-
-@BelongsTo(parent = ResourceGroups.class, foreignKeyName = "resource_group_id")
-@IdName("resource_group_id")
-@Table("selectors") // located in gateway-ha-persistence-*.sql
-@Cached
-public class Selectors
-        extends Model
+import org.jdbi.v3.core.mapper.reflect.ColumnName;
+
+public record Selectors(
```

Comment:
Dont we also want a  @JdbiConstructor in this class?

Reply:
https://jdbi.org/#_constructormapper
`@JdbiConstructor` is required when there are multiple constructors.

Reply:
oh .. thanks for explaining.